recipes/causalml/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "causalml" %}
+{% set version = "0.12.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/causalml-{{ version }}.tar.gz
+  sha256: 4d85afa0590e4c60818311c2be9a5be64b8c6964b134c77654b738df551454fa
+
+build:
+  number: 0
+  skip: true   # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest
+    - python
+    - cython
+    - setuptools >=41.0.0
+    - numpy
+    - scikit-learn >=0.22.0
+    - python-graphviz
+    - graphviz
+  run:
+    - python
+    - setuptools >=41.0.0
+    - pip >=10.0
+    - numpy
+    - scipy >=1.4.1
+    - matplotlib-base
+    - pandas >=0.24.1
+    - scikit-learn >=0.22.0
+    - statsmodels >=0.9.0
+    - seaborn
+    - cython
+    - xgboost
+    - pydotplus
+    - tqdm
+    - shap
+    - dill
+    - lightgbm
+    - pygam
+    - packaging
+    - pytorch
+    - pyro-ppl
+    - python-graphviz
+    - graphviz
```

Comment:
I don't think you need the `graphviz` c library do you? If `python-graphviz` needs it, the package will pull it in itself.